### PR TITLE
[stable/postgresql] Allow mounting secrets as files

### PR DESCRIPTION
**Pull request by javsalgar**
Signed-off-by: Javier J. Salmeron Garcia <jsalmeron@bitnami.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by juan131 — CHANGES_REQUESTED** [review]
Please rebase you branch with latest changes in master

File: stable/postgresql/templates/statefulset-slaves.yaml [hunk 1]
```diff
@@ -135,6 +136,10 @@ spec:
           failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
         {{- end }}
         volumeMounts:
+        {{- if .Values.usePasswordFile }}
+        - name: postgresql-password
+          mountPath: /opt/bitnami/postgresql/secrets
+        {{ end }}
```
Comment (juan131) [inline stable/postgresql/templates/statefulset-slaves.yaml:140]: Should we mount it as `readOnly`?

```
{{- if .Values.usePasswordFile }}
- name: postgresql-password
  mountPath: /opt/bitnami/postgresql/secrets
  readOnly: true
{{ end }}
```

File: stable/postgresql/templates/statefulset.yaml [hunk 2]
```diff
@@ -151,6 +157,10 @@ spec:
         volumeMounts:
         - name: custom-init-scripts
           mountPath: /docker-entrypoint-initdb.d
+        {{- if .Values.usePasswordFile }}
```
Comment (juan131) [inline stable/postgresql/templates/statefulset.yaml:160]: Same concern about `readOnly`

File: stable/postgresql/templates/statefulset-slaves.yaml (hunk 1 above)
Comment (javsalgar) [reply stable/postgresql/templates/statefulset-slaves.yaml:140]: This is done by default when we are using secrets and configmaps

Comment (juan131) [reply stable/postgresql/templates/statefulset-slaves.yaml:140]: 👍

**Comment by juan131** [comment]
You need to sign-off you commits

**Pull request merged**